### PR TITLE
Use data dir flag

### PR DESCRIPTION
**Pull request by felipemadero**
This is just a first step on https://github.com/ava-labs/avalanche-network-runner/issues/400 to quickly enable users to have all the node dirs in one separate location, While the bigger update is on the way

### CODE REVIEW

File: local/network.go [hunk 1]
```diff
@@ -965,6 +971,7 @@ func (ln *localNetwork) buildFlags(
 	// Flags for AvalancheGo
 	flags := []string{
 		fmt.Sprintf("--%s=%d", config.NetworkNameKey, ln.networkID),
+		fmt.Sprintf("--%s=%s", config.DataDirKey, dataDir),
```
Comment (dasconnor) [inline local/network.go:974]: Bad formatting

**Review by dasconnor — APPROVED** [review]
LGTM

File: local/network.go (hunk 1 above)
Comment (felipemadero) [reply local/network.go:974]: which one? don't understand

Comment (dasconnor) [reply local/network.go:974]: nvm, this displayed incorrectly on my phone

**Pull request merged**